Sources/LoggingTelegram/TelegramLogHandler.swift

```diff
@@ -60,10 +60,11 @@ public class TelegramLogHandler<T>: LogHandler where T: TelegramId {
 
   /// Creates a `TelegramLogHandler` for sending `Logger` output directly to Telegram.
   /// - Parameters:
-  ///   - label: The log label for he log handler.
-  ///   - chats: Target chats to send.
+  ///   - label: The log label for the log handler.
+  ///   - chat: Target chats to send to.
```

Comment:
`chats` is also a typo here, you may fix it at the same time.

Reply:
Oh I'll fix it.

> `chats` is also a typo here, you may fix it at the same time.